examples/dataflow-xml-pubsub-to-gcs/python/.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Add license here as well please.

Reply:
Addressed in latest commit, please review